package.json

```diff
@@ -192,7 +196,9 @@
     "node-polyglot": "2.3.1",
     "number-to-locale-string": "1.2.0",
     "piwik-react-router": "0.12.1",
+    "playwright-core": "^1.0.1",
     "pouchdb-adapter-cordova-sqlite": "2.0.5",
+    "qawolf": "^1.0.1",
```

Comment:
why having qawolf / playwright in both dep and dev-dep? 

Should argos-cli be a dev-dep also?

Reply:
I think its qawolf that installed playwright, anyway I put everything in devDeps, good catch.